File: 240416/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
a = ["apple", "banana", "grape", "blueberry", "orange"]
b = input()
count = 0

for i in range(len(a)):
    if (a[i][2] == b) or (a[i][3] == b):
        print(a[i])
        count += 1
print(count)

File: 241226/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int cnt = 0;
    string arr[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    char c;
    cin >> c;
    for (int i=0; i<5; i++) {
        if (arr[i][2] == c || arr[i][3] == c) {
            cout << arr[i] << endl;
            cnt++;
        } 
    }
    cout << cnt;

    return 0;
}